source/chapters/appendix/keyboard_mapping_table.rst

```diff
@@ -132,7 +132,11 @@ language settings the defaults may be different.
 +----------------------------------------+-----------------------------------------+------------------------------------------+
 | Clear search input                     | :kbd:`Ctrl` + :kbd:`Back`                                                          |
 +----------------------------------------+-----------------------------------------+------------------------------------------+
-| Exit search and leave focus            | :kbd:`ESC`                                                                         |
+| Exit search and leave focus            | :kbd:`Escape`                                                                      |
```

Comment:
while you're at it:
Ctrl + F is a global shortcut set in wtracktableview. cpp, so this should actually be
"Leave current widget and focus tracks table"

Reply:
Apart from that thos LGTM

Reply:
> "Leave current widget and focus tracks table"

This text does not fit in this table. Should I (just) enlarge the entire table?

Reply:
Well, just "Focus tracks table" is sufficient.

We may improve the documentaion later on, for example add `Tab`